github/Requester.py

```diff
@@ -411,10 +411,20 @@ def __requestRaw(self, cnx, verb, url, requestHeaders, input):
             return self.__requestRaw(original_cnx, verb, url, requestHeaders, input)
 
         if status == 301 and 'location' in responseHeaders:
-            return self.__requestRaw(original_cnx, verb, responseHeaders['location'], requestHeaders, input)
+            redirect_url = self.__permanentRedirectURL(responseHeaders['location'], cnx)
```

Comment:
The problem that I've seen:
- cnx.protocol: https
- cnx.host: github.enterprise.com
- responseHeaders['location']: https://github.enterprise.com/api/